### ISSUE TITLE
Expect float: Google/Protobuf/DoubleValue.php(61) when fetching 'metrics.all_conversions_value' 

### ISSUE THREAD

**Issue by joelbutcher** [protocolbuffers/protobuf-php#4]
I'm getting an `Expect float` error which traces back to `setAllConversionsValue` in the `Metrics.php` from Google's Ads v2.0.

```Expect float. {"trace":"#0 /Users/joelbutcher/Adzooma/adzooma/vendor/google/protobuf/src/Google/Protobuf/DoubleValue.php(61): Google\\Protobuf\\Internal\\GPBUtil::checkDouble(Object(Google\\Protobuf\\DoubleValue))
#1 /Users/joelbutcher/Adzooma/adzooma/vendor/google/protobuf/src/Google/Protobuf/Internal/Message.php(1009): Google\\Protobuf\\DoubleValue->setValue(Object(Google\\Protobuf\\DoubleValue))
#2 /Users/joelbutcher/Adzooma/adzooma/vendor/google/protobuf/src/Google/Protobuf/Internal/Message.php(79): Google\\Protobuf\\Internal\\Message->mergeFromArray(Array)
#3 /Users/joelbutcher/Adzooma/adzooma/vendor/google/protobuf/src/Google/Protobuf/DoubleValue.php(38): Google\\Protobuf\\Internal\\Message->__construct(Array)
#4 /Users/joelbutcher/Adzooma/adzooma/vendor/googleads/google-ads-php/src/Google/Ads/GoogleAds/V1/Common/Metrics.php(1843): Google\\Protobuf\\DoubleValue->__construct(Array)
#5 /Users/joelbutcher/Adzooma/adzooma/vendor/google/protobuf/src/Google/Protobuf/Internal/Message.php(489): Google\\Ads\\GoogleAds\\V1\\Common\\Metrics->setAllConversionsValue(Object(Google\\Protobuf\\DoubleValue))```

**Comment by TeBoring** [comment]
This is a name conflict with the helper method of wrapper messages in ads
proto file. We solved it in 3.9.0-rc1 by renaming getXXXValue to
getXXXUnwrapped for wrapper messages.

**Comment by TeBoring** [comment]
Ads will soon update their generated code.
